app/controllers/dashboard_controller.rb

```diff
@@ -14,6 +14,16 @@ def index
     end
   end
 
+  def show
+    if @widget.present? && @widget.user == User.current
+      render(:partial => @widget.template, :locals => @widget.data)
```

Comment:
Did you restart the rails server? it seems to work fine for me (tested with tasks plugin).
Also, the widgets are logged, e.g. 
```
2016-08-23T10:51:59 a9570e1f [app] [I] Started GET "/widgets/16" for 127.0.0.1 at 2016-08-23 10:51:59 +0300
2016-08-23T10:51:59 a9570e1f [app] [I] Processing by DashboardController#show as HTML
2016-08-23T10:51:59 a9570e1f [app] [I]   Parameters: {"id"=>"16"}
2016-08-23T10:52:00 a9570e1f [app] [I]   Rendered dashboard/_distribution_widget.html.erb (19.5ms)
2016-08-23T10:52:00 a9570e1f [app] [I] Completed 200 OK in 514ms (Views: 16.5ms | ActiveRecord: 235.1ms)